src/functionalTest/resources/features/F-002 - Get Assignments in My Organisation/S-008.td.json

```diff
@@ -0,0 +1,70 @@
+{
+  "_guid_": "S-008",
+
+  "title": "Must return case assignments in my organisation for the provided Case IDs",
+
+  "specs": [
+    "Richard – a solicitor with the required permissions to create a case",
+    "Dil – another solicitor within the same organisation as Richard",
+    "Jamal – another solicitor within the same organisation as Richard",
+    "is made by Richard",
+    "contains a correctly-formed comma separated list of the valid case ID’s of C1 and C2",
+    "includes the respective case assignments of user ID's of both Dil and Jamal and cases C1 and C2"
+  ],
+
+  "productName": "Manage Case Assignment Microservice",
+  "operationName": "Get Assignments in My Organisation",
+
+  "s2sClientId": "xui_webapp",
+  "userTokenClientId": "xuiwebapp",
+
+  "method": "GET",
+  "uri": "/case-assignments",
```

Comment:
These standard attributes should be reused from a common base object of this Feature, and then these files should be showing only what is specific to them.

Reply:
Please apply the same comment in every place where data reuse could be enhanced.

Reply:
👍